app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -3369,6 +3377,32 @@ class BrowserTabFragment :
         daxDialogIntroBubble.root.gone()
     }
 
+    private var proxy: JavaScriptReplyProxy? = null
+
+    private val delay = "\$DELAY$"
+    private val postInitialPing = "\$POST_INITIAL_PING$"
+    private val replyToNativeMessages = "\$REPLY_TO_NATIVE_MESSAGES$"
+
+    private fun configureWebViewForWebViewCompatTest(webView: DuckDuckGoWebView) {
```

Comment:
can we move all of this to its own class to keep `BTF` changes minimal and contained? thinking there would be a single line in `BTF` which calls through to your thing passing the `WebView` for instance.